src/javascript/JContent/ContentRoute/ContentLayout/FilesGrid/FileCard/FileCard.jsx

```diff
@@ -159,86 +159,84 @@ export const FileCard = ({gridMode,
     let encodedPath = node.path.replace(/[^/]/g, encodeURIComponent);
     let isPdf = node.children ? node.children.nodes.filter(node => node.mimeType.value === 'application/pdf').length !== 0 : false;
     return (
-        <React.Fragment>
+        <Card
+                style={{msGridColumn: columnNumber, msGridRow: rowNumber}}
+                className={classNames(
+                    classes.card,
+                    isThumbCard && classes.thumbCard,
+                    isDetailedCard && classes.detailedCard,
+                    isPreviewSelected && classes.selectedCard
+                )}
+                data-cm-role="grid-content-list-card"
+                onContextMenu={event => {
+                    event.stopPropagation();
+                    contextualMenu.current(event);
+                }}
+                onClick={() => {
+                    if (!node.notSelectableForPreview) {
+                        onPreviewSelect(node.path);
+                    }
+                }}
+                onDoubleClick={allowDoubleClickNavigation(node.primaryNodeType.name, null, () => setPath(siteKey, node.path, mode))}
+        >
             <ContextualMenu setOpenRef={contextualMenu} actionKey="contentMenu" path={node.path}/>
```

Comment:
The diff is not easily readable...
I basically just moved the <ContextualMenu> component inside the <Card> one.